test/tools/HttpsListener/Pages/Index.cshtml.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Text;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.AspNetCore.Server.Kestrel.Https;
+
+namespace HttpsListener.Pages
+{
+    public class IndexModel : PageModel
+    {
+        public string CertSubject;
+        public string CertNotAfter;
+        public string CertNotBefore;
+        public string CertIssuer;
+        public string CertIssuerName;
+        public string CertSubjectName;
+        public string CertThumbprint;
+        public string Status = "FAILED";
```

Comment:
My understanding is that it does in order to be referenced in the view. I can test with private though.